.circleci/config.yml

```diff
@@ -16,6 +16,9 @@ workflows:
       - test-ios:
           requires:
             - validate-code
+      - test-ios-integration:
```

Comment:
Having it in parallel with the tests helps save time, but I am thinking if we should run it after the unit and functional test, so in case those fail we don't run these integration tests. This would help with saving costs and unnecessary e2e run. This may not add much value since we are running on the staging, so ideally all our unit and functional tests would be passing in the dev branch already, but just a thought.